Jetsnack/app/src/main/java/com/example/jetsnack/ui/home/cart/Cart.kt

```diff
@@ -185,104 +202,217 @@ fun CartItem(
     modifier: Modifier = Modifier
 ) {
     val snack = orderLine.snack
-    ConstraintLayout(
-        modifier = modifier
+    var offsetX by remember { mutableStateOf(0f) }
+    Box(
```

Comment:
Could this use the [`SwipeToDismiss` composable](https://cs.android.com/androidx/platform/frameworks/support/+/androidx-main:compose/material/material/src/commonMain/kotlin/androidx/compose/material/SwipeToDismiss.kt;l=172)?